lib/common/CMakeLists.txt

```diff
@@ -43,7 +43,8 @@ set (HEADERS
 	${PROJECT_SOURCE_DIR}/include/vapor/Base16StringStream.h
 )
 
-add_library (common SHARED ${SRC} ${HEADERS})
+find_library(HDF5_LIB hdf5)
+add_library (common SHARED ${SRC} ${HEADERS} ${HDF5_LIB})
```

Comment:
Libraries aren't valid arguments to `add_library`

Reply:
You're correct.  Fixed.